datalad/cmdline/helpers.py

```diff
@@ -433,7 +433,11 @@ def _maybe_get_single_subparser(cmdlineargs, parser, interface_groups,
         if not unparsed_args and getattr(parsed_args, 'version', None):
             parsed_args.version()  # will exit with 0
         if not (completing or unparsed_args):
-            fail_handler(parser, msg="too few arguments", exit_code=2)
+            fail_handler(
+                parser,
+                msg="too few arguments, run with --help to get more information.\n"
+                    "visit: https://www.datalad.org https://docs.datalad.org https://handbook.datalad.org",
```

Comment:
This is a user-facing message intended to aid datalad usage. I think it is important to keep it as short as possible.

https://www.datalad.org has infos on what datalad is (at best), users that see this message already have it.

https://docs.datalad.org has almost exclusively technical information, users do not need that, when they still have no idea how to run it.

That let's me conclude that only the last pointer should retained, and the first two stripped for clarity and conciseness. The handbook website does have links to the project website and the technical docs on the frontpage too.

Reply:
Ok, I will drop 2nd commit

The idea was to provide extra information (potentially long but fitting into 80x40) for completely unparametrized invocation `datalad` (meaning that user really didn't know how to use datalad) but I guess it should indeed not be in this handler which might be triggered in other situations too

Reply:
And for such situations IMHO it would be useful to provide useful URLs which would describe what it is, have commands reference and provide tutorials etc

Reply:
Ah, sorry, a misunderstanding! I am very much in favor of giving a URL -- just not 3 URLs at once.